brave/brave-core
# Remove BAP UI and bitFlyer feature flag

Resolves https://github.com/brave/brave-browser/issues/15342

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Navigate to `brave://flags` and verify that "Enable bitFlyer for Brave Rewards" is not available.
* Verify that JP users can connect a bitFlyer wallet.
